### ISSUE TITLE
MNT: Remove deprecated introspection.isinstancemethod() and introspection.find_mod_objs()

### ISSUE THREAD

**Issue by pllim** [astropy/astropy#15936]
Deprecated as of v6.1 , so removal must be v7.1 or later, so any PR for this cannot be merged before v7.0.x branching happens.

* https://github.com/astropy/astropy/pull/15934

**Comment by GuilhermePonce** [comment]
Hey pllim, I wanna try to solve this issue even though it will only be accepted when v7.0x happens.

**Comment by pllim** [comment]
Hello and thanks for your interest. You can always open a draft PR for now if you insist on working on this issue. Though we also have others available; your choice.

**Comment by neutrinoceros** [comment]
> so removal must be v7.1 or later

I don't think we can (or should) remove anything in a minor release. Did you mean 8.0 or later ? this would also mean that we can't merge a closing PR before 8.0 is branched out.

**Comment by pllim** [comment]
Oh... maybe. I cannot keep track of all the APE updates. Thx for the reminder. I will update original post.

**Comment by NathanMuniz18** [comment]
hello guys, I would like to help with this issue and would like to know if I have permission to contribute to it?

**Comment by jomigmonte** [comment]
Hello, I would like to know if its possible to work in this issue now in version 8.0

**Comment by pllim** [comment]
Thanks for your interest. Looks like I forgot about this again and unfortunately we're too late for 8.0 so this will have to wait a bit more for 9.0.

**Comment by jomigmonte** [comment]
I understand. Even so, can I make a draft PR that adresses this issue? Even if it doesn't get merged.

**Comment by pllim** [comment]
> Even so, can I make a draft PR that adresses this issue

Yes. This is open source. No one can stop you if you really want to do it. 😹

**Comment by pllim** [comment]
Hmm now I have confused myself. Original post said 8.0 or later, but I milestoned this 9.0. Why, Past Self? ™️

**Comment by pllim** [comment]
Oh right... it missed the 8.0 freeze... nvm 🤦‍♀️
